src/bigdecimal.ts

```diff
@@ -2956,6 +2967,101 @@ export class BigDecimal {
         }
     }
 
+    /**
+    * Alias for `compareTo(val) === 0`.
+    * Consider using {@link equals} in case the precision needs to be considered.
```

Comment:
```suggestion
    * Consider using {@link equals} in case the scale needs to be considered.
```

I think scale differs between 2.00 and 2.0, not precision.

Reply:
Fixed in b9e0c777a81f9fc9283be03ed134960ef9dee293